code/game/gamemodes/objective.dm

```diff
@@ -795,6 +797,31 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 		else
 			return FALSE
 
+/datum/objective/paradox_replace
+	name = "Replacement"
+	martyr_compatible = FALSE
+
+/datum/objective/paradox_replace/update_explanation_text()
+	if(target?.current)
+		explanation_text = "Assassinate [target.current.real_name], the [target.assigned_role] and replace them! Use 'Replace' ability to erase their body! Don't forget, your mission is to blend up, try to don't kill anyone else."
```

Comment:
```suggestion
		explanation_text = "Assassinate [target.current.real_name], the [target.assigned_role] and replace them! Use your 'Replace' ability to erase their body! Don't forget, your mission is to blend in, so try to avoid collateral damage."
```